deepfence_frontend/apps/dashboard/src/components/remediation/RemediationNoIntegration.tsx

```diff
@@ -22,6 +25,37 @@ export const RemediationNoIntegration = ({
         <DFLink to="/integrations/gen-ai/add">Add Generative AI integrations</DFLink> page
         to add one now!
       </div>
+      {state === 'idle' && !data && (
+        <div className="text-center text-p2 mt-2 px-6">
+          If the management console is deployed in AWS with instance role and read
+          permission to Amazon Bedrock, please{' '}
+          <button
+            type="button"
+            className="text-blue-600 dark:text-text-link hover:underline focus:underline"
+            onClick={(e) => {
+              e.preventDefault();
+              add();
+            }}
+          >
+            click here
+          </button>{' '}
+          to automatically add integrations.
+        </div>
+      )}
+      {state === 'idle' && data?.success === false && (
+        <div className="text-center text-p2 mt-2 px-6 dark:text-status-error">
+          {data.message ?? "Couldn't add integration."}
+        </div>
+      )}
+      {state === 'idle' && data?.success && (
+        <div className="text-center text-p2 mt-2 px-6 dark:text-status-success">
+          {data.successMessage ?? 'Successfully added integration.'}
+        </div>
```

Comment:
For uniformity, can we update text 'Successfully added integration.' to 'Integration added successfully.'?